server/ntb/scanpix/scanpix_datalayer.py

```diff
@@ -213,7 +213,7 @@ def _parse_doc(self, doc):
             new_doc['original_source'] = new_doc['source'] = doc['credit']
         except KeyError:
             pass
-        new_doc['versioncreated'] = utcnow()
+        new_doc['versioncreated'] = self._datetime(doc['archivedTime'])
         new_doc['firstcreated'] = self._datetime(doc['archivedTime'])
```

Comment:
these 2 lines could be merged to avoid calling 2 times the same method: 
  new_doc['versioncreated'] = new_doc['firstcreated'] = self._datetime(doc['archivedTime'])